atm.js

```diff
@@ -1,21 +1,69 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+$(document).ready(function(){
```

Comment:
the `document.ready` isn't wrong, but is unecessary here because your script tag is loaded after the DOM has loaded because it's at the bottom of the page and not in the head.